### PR TITLE
faeture:users management(show,delete)

### PR DESCRIPTION
**Pull request by Kovalenko77**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+let globalUsers = [];
+const SERVER_URL = 'https://jsonplaceholder.typicode.com';
```
Comment (romanalesenkov1) [inline index.js:2]: add empty line below

File: index.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+init();
+
+function init() {
+  return getAllusers();
+}
+
+function getAllusers() {
```
Comment (romanalesenkov1) [inline index.js:244]: use camelCase for function names

File: index.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    .finally(res=> console.log('finally'));
+}
+
+function showAllUsers(users) {
+  const usersHtmlEl = document.getElementById('users');
+  usersHtmlEl.innerHTML = users.map((user, index)=>{return getInfo(user, index)} ).join('');
+}
+
+function getInfo(user, index) {
+  return `<div id='${user.id}'>
```
Comment (romanalesenkov1) [inline index.js]: id is an id of an element.
Please, use data-id instead of id here.

File: index.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+   <button onclick='getMoreInfo(event)'>more info</button> |
+   <button onclick='deleteUser(event)'>delete</button> 
+   </div> `;
+}
+
+function getMoreInfo(event) {
+  const activatedButton = event.target;
+  const userHtmlEl = activatedButton.parentElement;
+  const userId = userHtmlEl.id;
+  fetch(SERVER_URL +'/users/'+ userId)
```
Comment (romanalesenkov1) [inline index.js]: add spaces

File: index.js [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+  const userId = userHtmlEl.id;
+  fetch(SERVER_URL +'/users/' + userId , {
+    method: 'DELETE',
+  })
+  .then((response)=> {
+    globalUsers = globalUsers.filter((user)=> {return user.id !== Number(userId)});
+    console.warn('user has been successfully deleted');
+    showAllUsers(globalUsers);
+  })
+  .catch(error=> console.error('Error of deletting users'))
```
Comment (romanalesenkov1) [inline index.js]: deleting

File: index.html [hunk 6]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="style.css" rel="stylesheet" type="text/css">
```
Comment (romanalesenkov1) [inline index.html]: why do you link an inexistent css file?

**Review by romanalesenkov1 — COMMENTED** [review]
review

File: index.js (hunk 1 above)
Comment (Kovalenko77) [reply index.js:2]: Done. Thanks.

File: index.js (hunk 3 above)
Comment (Kovalenko77) [reply index.js]: Сorrected. Thanks.

File: index.js (hunk 4 above)
Comment (Kovalenko77) [reply index.js]: Corrected. Thanks.

File: index.js (hunk 5 above)
Comment (Kovalenko77) [reply index.js]: Corrected. Thanks.

**Pull request merged**